static/js/statPanel.js

```diff
@@ -14,13 +15,25 @@ function statPanelClick(widthOfPanel, heightOfPanel, chartWidth, chartHeight) {
 	//callback: startJSPanelWithChart(jsPanel, widthOfPanel, heightOfPanel, chartWidth, chartHeight)
 	callback: function (jsPanel) {
 		jsPanelInst = jsPanel;
-		startJSPanelWithChart( jsPanel, widthOfPanel, heightOfPanel, chartWidth, chartHeight );
+		startJSPanelWithChart( jsPanel, widthOfPanel, heightOfPanel, chartWidth, chartHeight);
 		}
 	});
 };
 
-var startJSPanelWithChart = function( jsPanel, widthOfPanel, heightOfPanel, chartWidth, chartHeight ){
-	var str = '<div id="myChart" width="';
+var startJSPanelWithChart = function( jsPanel, widthOfPanel, heightOfPanel, chartWidth, chartHeight){
+	var str = '<select class="form-control" id="selectTypeOfChart">'
+    if (currentTypeOfChart==="accidentsPerMonth"){
+        str+= '<option value="accidentsPerMonth" selected>מספר התאונות בחודש על פני כל השנים שנבחרו</option>';
```

Comment:
Any chance to avoid the repetition? (For example, use a jQuery selector to set `selected` only on the option whose `value` equals `currentTypeOfChart`)